index.html

```diff
@@ -33,5 +33,6 @@ <h2 id="monthText"></h2>
     <img src="" id ="cartoon">
 
     <script type="text/javascript" src="/js/main.js"></script>
+    <script src="/js/ghostCursor.js"></script>
```

Comment:
It looks like you didn't pull the most recent changes made to server.js, so this won't be read. Currently, your server.js file only has an fs.readFile for main.js, not this one, so we will always get the 404 page anytime this loads. The more recent changes though are updated to take into account other js files